src/main/java/nl/opengeogroep/safetymaps/server/stripes/SafetyConnectProxyActionBean.java

```diff
@@ -70,56 +83,143 @@ public Resolution proxy() throws Exception {
             return new ErrorMessageResolution(HttpServletResponse.SC_INTERNAL_SERVER_ERROR, "Geen toegangsgegevens voor webservice geconfigureerd door beheerder");
         }
 
-        // TODO: serverside checks incidentmonitor, incidentmonitor_kladblok,
-        // eigen_voertuignummer roles and use voertuignummer from user details
-
-        String qs = context.getRequest().getQueryString();
-        final HttpUriRequest req = RequestBuilder.get()
-                .setUri(url + "/" + path + (qs == null ? "" : "?" + qs))
-                .addHeader("Authorization", authorization)
-                .build();
-
-        try(CloseableHttpClient client = HttpClients.createDefault()) {
-            final MutableObject<String> contentType = new MutableObject<>("text/plain");
-            final String content = client.execute(req, new ResponseHandler<String>() {
-                @Override
-                public String handleResponse(HttpResponse hr) {
-                    log.debug("proxy for user " + context.getRequest().getRemoteUser() + " URL " + req.getURI() + ", response: " + hr.getStatusLine().getStatusCode() + " " + hr.getStatusLine().getReasonPhrase());
-                    contentType.setValue(hr.getEntity().getContentType().getValue());
-                    try {
-                        return IOUtils.toString(hr.getEntity().getContent(), "UTF-8");
-                    } catch(IOException e) {
-                        log.error("Exception reading HTTP content", e);
-                        return "Exception " + e.getClass() + ": " + e.getMessage();
+        try(Connection c = DB.getConnection()) {
```

Comment:
Note that this keeps a db connection open during proxying. If the SC server happens to be slow to respond or the load is very high, this might exhaust the connection pool which is usually set to 40 concurrent connections max. 

Assuming the `Cfg.getSettings()` call does not open a DB connection because the settings are cached, the only call which needs a DB connection is `getUserDetails()`, which is only required after the response has been received.